nixos/common/nix.nix

```diff
@@ -9,9 +9,17 @@
     "flakes"
     "repl-flake"
     "impure-derivations"
+    # for container in builds support
     "auto-allocate-uids"
+    "cgroup"
   ];
 
+  # no longer need to pre-allocate build users for everything
+  nix.settings.auto-allocate-uids = true;
```

Comment:
Can I ask a stupid question: Would I need to set use-cgroups, too? When running nix show-config it showed as being disabled but auto-allocate-uids mentions that it requires this feature and it worked without it for me.